File: test-service-java/src/main/java/com/example/testservicejava/v1/repositories/GeoRepository.java
package com.example.testservicejava.v1.repositories;

import com.example.testservicejava.v1.entities.Geo;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface GeoRepository extends JpaRepository<Geo, Long> {}

File: test-service-kotlin/src/main/kotlin/org/xapps/service/testservice/v1/repositories/GeoRepository.kt
package org.xapps.service.testservice.v1.repositories

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import org.xapps.service.testservice.v1.entities.Geo

@Repository
interface GeoRepository : JpaRepository<Geo, Long>